charts/incubator/minecraft-bungeecord/values.yaml

```diff
@@ -27,7 +27,7 @@ secretEnv:
   RCON_PASSWORD: "secretrconpass"
 
 env:
-  RCON_PORT: "{{ .Values.service.rcon.ports.rcon.port }}"
+  RCON_PORT: "{{ .Values.service.rcon.ports.rcon.targetPort }}"
```

Comment:
Why?!

As I already said, this was fine. No need to change.

Any port number you set on `port` is set to both `targetPort` and in the `RCON_PORT` environment variable.
Which changes what port the app is listening.

Reply:
how does it set the target port? that was removed a few PRs ago.

Reply:
![stavs-pr](https://user-images.githubusercontent.com/9399967/197277757-68ad5757-4246-47e8-9e47-738731f46e39.png)

Reply:
https://github.com/truecharts/library-charts/blob/f267828efef0c2e1fc0f22f7be0de114c5bc3387/charts/common/templates/lib/controller/_ports.tpl#L23

Defaults to `port` if `targetPort` is not defined

Reply:
....that explains it., ill fix it now.